[BUG] recent occasional UT failure

```
 =========================== short test summary info ============================
FAILED source/tests/pt/model/test_trans.py::TestEnergyModelHybrid::test - AssertionError: Tensor-likes are not close!

Mismatched elements: 1 / 9 (11.1%)
Greatest absolute difference: 1.9291310085836622e-10 at index (0,) (up to 1e-10 allowed)
Greatest relative difference: 5.051167175729902e-08 at index (0,) (up to 1e-10 allowed)
= 1 failed, 719 passed, 901 skipped, 8096 deselected, 124 warnings in 408.48s (0:06:48) =
```

```
         for key in test_keys:
            if key in ["energy", "force", "force_mag"]:
                torch.testing.assert_close(ret0[key], ret1[key], rtol=prec, atol=prec)
            elif key == "virial":
                if not hasattr(self, "test_virial") or self.test_virial:
>                   torch.testing.assert_close(
                        ret0[key], ret1[key], rtol=prec, atol=prec
                    )
E                   AssertionError: Tensor-likes are not close!
E                   
E                   Mismatched elements: 1 / 9 (11.1%)
E                   Greatest absolute difference: 1.9291310085836622e-10 at index (0,) (up to 1e-10 allowed)
E                   Greatest relative difference: 5.051167175729902e-08 at index (0,) (up to 1e-10 allowed)

source/tests/pt/model/test_trans.py:73: AssertionError
```

from https://github.com/deepmodeling/deepmd-kit/actions/runs/9276252769/job/25522908653?pr=3829

```
FAILED source/tests/pt/model/test_polarizability_fitting.py::TestEquivalence::test_rot - AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 27 / 45 (60%)
Max absolute difference: 1.02152152e-05
Max relative difference: 0.00697244
 x: array([[[[ 7.400039e-04,  6.783456e-05, -7.617659e-04],
         [ 6.783456e-05,  1.840847e-05, -1.378119e-04],
         [-7.617659e-04, -1.378119e-04,  1.121033e-03]],...
 y: array([[[[ 7.400039e-04,  6.783456e-05, -7.617659e-04],
         [ 6.783456e-05,  1.840847e-05, -1.378119e-04],
         [-7.617659e-04, -1.378119e-04,  1.121033e-03]],...
= 1 failed, 721 passed, 895 skipped, 8085 deselected, 122 warnings in 435.29s (0:07:15) =
```

from https://github.com/deepmodeling/deepmd-kit/actions/runs/9287263428/job/25563045054?pr=3837

--- a/source/tests/pt/model/test_trans.py
+++ b/source/tests/pt/model/test_trans.py
@@ -64,7 +64,7 @@ class TransTest:
             spins=spin.unsqueeze(0),
         )
         ret1 = {key: result_1[key].squeeze(0) for key in test_keys}
-        prec = 1e-10
+        prec = 1e-7
         for key in test_keys:
             if key in ["energy", "force", "force_mag"]:
                 torch.testing.assert_close(ret0[key], ret1[key], rtol=prec, atol=prec)

